reana_client/config.py

```diff
@@ -34,12 +34,12 @@
 default_user = '00000000-0000-0000-0000-000000000000'
 """Default user to use when submitting workflows to Reana Server."""
 
-default_organization = 'default'
-"""Default organisation to use when submitting workflows to Reana Server."""
-
 ERROR_MESSAGES = {
         'missing_access_token':
         'Please provide your access token by using'
         ' the -at/--access-token flag, or by setting the'
         ' REANA_ACCESS_TOKEN environment variable.'
 }
+
+default_download_path = './outputs/'
```

Comment:
It is unused right now (removed in the outputs/inputs/code unification). However, I though that you plugged it again for safety (which actually makes sense) otherwise the file will be downloaded in current directory potentially overwriting important files. @dinosk WDYT about plugging it again? It is just that since we went to a totally plain workspace I did it like this.

Reply:
I saw that it gets downloaded to the current directory indeed, maybe we could add an optional parameter download_path, allowing the user to do:
```
reana-client files download plot.png outputs/myanalysis2
```

Either way, an issue should be opened about this and the fix be done in a separate PR, and I'll remove the config var here